src/js/iotjs.js

```diff
@@ -76,13 +76,13 @@
       if (/.*:/.test(line)) {
         file = line.slice(0, -1);
       } else {
-        var m = line.match(/(\+ ([a-zA-Z0-9_]*))? \[(\d+),(\d+)\] (\d+)/);
+        var m = line.match(/(\+ ([a-zA-Z0-9_]*))?( \[(\d+),(\d+)\])? (\d+)/);
```

Comment:
Top level source has no function name. Line in dump file is just like
```
 12345
```
So [line,column] should be optinal.